### ISSUE TITLE
Firacode is licensed under OFL but license and copyright statement is missing

### ISSUE THREAD

**Issue by papojari** [ejmg/zerm#59]

**Comment by ejmg** [comment]
good catch. i /think/ i did include it initially, but i may have accidentally removed it during the initial setup of this theme.

you are free to add a copy yourself via a PR, otherwise i add it myself soon.
